### PR TITLE
MM-53274 - Add a Beta system console setting for Calls ringing

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Add a Beta system console setting for Calls ringing, because the feature should be considered Beta (or even experimental) until probably 8.1 (which is when it should be released for Mobile as well).
- Default off, until later.

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-53274

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -160,8 +160,14 @@
...
         "hosting": "on-prem"
-      }
-    ]
+      },
+      {
+        "key": "EnableRinging",
+        "display_name": "Enable call ringing (Beta)",
+        "type": "bool",
+        "default": false,
+        "help_text": "When set to true it enables the call ringing functionality: calls started in DM or GM channels will ring for other members of the channel."
```
Comment (streamer45) [inline plugin.json:169]: Can we also add that they will send dedicated notifications?

**Review by streamer45 — APPROVED** [review]
Thanks, I am adding the "Docs needed" label as a reminder to update them before release time.

**Comment by streamer45** [comment]
@cpoile Heads up that I merged with `main` branch so that e2e should pass.

**Comment by cpoile** [comment]
Thanks, rebased (that was interesting, found a new use for `rebase --onto`).

**Pull request merged**